.github/ISSUE_TEMPLATE/01_bug_report.yml

```diff
@@ -29,15 +29,14 @@ body:
   attributes:
     label: Which MonoGame platform are you using?
     options:
+      - N/A
       - MonoGame Android Application (mgandroid)
       - MonoGame Cross-Platform Desktop Application (mgdesktopgl)
       - MonoGame Windows Universal CoreApp Application (mguwpcore)
       - MonoGame Windows Universal XAML Application (mguwpxaml)
       - MonoGame Windows Desktop Application (mgwindowsdx)
       - MonoGame iOS Application (mgios)
-      - MonoGame Game Library (mgnetstandard)
       - MonoGame Content Pipeline Extension (mgpipeline)
```

Comment:
Missed one library template.

Reply:
Missed it because it didn't say "library". 
(also as a side note, I use that template quite a lot for extension development, but I understand removing it from here)